_data/sites.json

```
@@ -2983,6 +2983,17 @@
         ]
     },
 
+    {
+        "name": "DaFONT",
+        "url": "https://www.dafont.com/forum/read/12643/how-do-i-delete-my-account",
+        "difficulty": "hard",
+        "notes": "You'll send a private message to Rodolphe : http://www.dafont.com/fr/profile.php?user=1 (Wrote by Vinz).",
```

What should this "Wrote by Vinz" mean?

---

Vinz (dafont user) wrote "You'll send a private message to Rodolphe" in dafont forum here: http://www.dafont.com/fr/profile.php?user=1

---

I see, in this case I don't think we need to add this information and can be a bit more succint.
```suggestion
        "notes": "Send a private message to Rodolphe: <a href='http://www.dafont.com/fr/profile.php?user=1'>.",
```